docs/admin/notifications/teams.md

```diff
@@ -0,0 +1,155 @@
+
+Administrators can configure Coder to send notifications via an incoming webhook
+endpoint. These notifications appear as messages in Teams chats, either with the
+Flow Bot or a specified user/service account.
+
+## Requirements
+
+Before setting up Microsoft Teams notifications, ensure that you have the
+following:
+
+- Administrator access to the Teams platform
+- Coder platform with notifications enabled
+
+## Build Teams Workflow
+
+The process of setting up a Teams workflow consists of three key steps:
+
+1.  Configure the Webhook Trigger.
+
+    Begin by configuring the trigger: **"When a Teams webhook request is
+    received"**.
+
+    Ensure the trigger access level is set to **"Anyone"**.
+
+2.  Setup the JSON Parsing Action.
+
+    Next, add the **"Parse JSON"** action, linking the content to the **"Body"**
+    of the received webhook request. Use the following schema to parse the
+    notification payload:
+
+    ```json
+    {
+    	"type": "object",
+    	"properties": {
+    		"_version": {
+    			"type": "string"
+    		},
+    		"payload": {
+    			"type": "object",
+    			"properties": {
+    				"_version": {
+    					"type": "string"
+    				},
+    				"user_email": {
+    					"type": "string"
+    				},
+    				"actions": {
+    					"type": "array",
+    					"items": {
+    						"type": "object",
+    						"properties": {
+    							"label": {
+    								"type": "string"
+    							},
+    							"url": {
+    								"type": "string"
+    							}
+    						},
+    						"required": ["label", "url"]
+    					}
+    				}
+    			}
+    		},
+    		"title": {
+    			"type": "string"
+    		},
+    		"body": {
+    			"type": "string"
+    		}
+    	}
+    }
+    ```
+
+    This action parses the notification's title, body, and the recipient's email
+    address.
+
+3.  Configure the Adaptive Card Action.
+
+    Finally, set up the **"Post Adaptive Card in a chat or channel"** action
+    with the following recommended settings:
+
+    **Post as**: Flow Bot
+
+    **Post in**: Chat with Flow Bot
+
+    **Recipient**: `user_email`
+
+    Use the following _Adaptive Card_ template:
+
+    ```json
+    {
+    	"$schema": "https://adaptivecards.io/schemas/adaptive-card.json",
+    	"type": "AdaptiveCard",
+    	"version": "1.0",
+    	"body": [
+    		{
+    			"type": "Image",
+    			"url": "https://coder.com/coder-logo-horizontal.png",
+    			"height": "40px",
+    			"altText": "Coder",
+    			"horizontalAlignment": "center"
+    		},
+    		{
+    			"type": "TextBlock",
+    			"text": "**@{replace(body('Parse_JSON')?['title'], '"', '\"')}**"
+    		},
+    		{
+    			"type": "TextBlock",
+    			"text": "@{replace(body('Parse_JSON')?['body'], '"', '\"')}",
+    			"wrap": true
+    		},
+    		{
+    			"type": "ActionSet",
+    			"actions": [@{replace(replace(join(body('Parse_JSON')?['payload']?['actions'], ','), '{', '{"type": "Action.OpenUrl",'), '"label"', '"title"')}]
```

Comment:
Can we add some explanation for why these `replace()` calls are neccessary?

Reply:
I can add an explanation below the snippet, as I'm not quite sure if Teams Workflow likes inline comments...

Reply:
Added 👍